cji_smoke_test.sh

```diff
@@ -61,6 +61,19 @@ if [ "$IQE_SELENIUM" = "true" ]; then
     SELENIUM_ARG=" --selenium "
 fi
 
+# check if there is a iqe-tests container image tag with the corresponding PR
+# number if not use the default IQE_IMAGE_TAG tag which you have to set in
+# order for this to work properly
+if [ ! -z "$ghprbPullId" ]; then
```

Comment:
Should this code also take into account gitlab MR's? The `ghprbPullId` only applies for jenkins jobs fired associated with github PR's... there's a different env var that is present in the case of a gitlab MR. For example: https://github.com/RedHatInsights/cicd-tools/blob/main/bootstrap.sh#L48C8-L55

Reply:
Ah, I was wondering what is that `$gitlabMergeRequestIid` for, so there are apps with code in Gitlab using the `pr_check.sh` scripts as well? In that case I yeah sure, I just hope it will work the same as I don't know how I would test it, our app is in Github only.